### PR TITLE
common: add document on running ArduPilot without RC inputs

### PR DESCRIPTION
**Pull request by peterbarker**
@Hwurzburg what do you think of this as a concept?

### CODE REVIEW

**Comment by proficnc** [comment]
Excellent. Quite a common configuration for swarming drones and delivery systems

**Comment by Hwurzburg** [comment]
I dont have any issue with this as long as there are enough caveats 
I would go into a LOT more detail of safe alternate control of the vehicle...ie a GCS is necessary and setup for and using a joystick for anything other than AUTO missions...GCS failsafes setup, etc... the exception being rover with a KICK or GO switch, but LOTs of caveats about that one!
-should link in the Advanced configurations TOC

File: common/source/docs/common-running-without-rc.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+========================================
+
+This article provides an overview of how ArduPilot can be configured
+to safely run with no RC receiver attached.
+
+.. note::
+
+   Fitting your vehicle with an RC receiver allows for additional recovery measures should your vehicle misbehave.  Particularly while creating and developing a vehicle we strongly recommend flying with an RC receiver/transmitter pair providing this additional recovery mechanism.
+
+  Before making the decision to configure your vehicle without an RC receiver, consider options such as ignoring RC failure for failsafe purposes; see "Configuring for Emergency RC Receiver Use", below.
```
Comment (nkruzan) [inline common/source/docs/common-running-without-rc.rst:14]: I don't see a section marked "Configuring for Emergency RC Receiver Use" below

**Comment by Hwurzburg** [comment]
that would work, only Rishab, Randy or I usually do that page.......as it stands, I cant get an output file from the above...maybe if its made into a standalone script generating an rst file in the directory from which its executed?

**Comment by nkruzan** [comment]
> that would work, only Rishab, Randy or I usually do that page.......as it stands, I cant get an output file from the above...maybe if its made into a standalone script generating an rst file in the directory from which its executed?

I think this is meant for #4087

**Comment by Hwurzburg** [comment]
yep too many open windows...moved

File: common/source/docs/common-running-without-rc.rst (hunk 1 above)
Comment (peterbarker) [reply common/source/docs/common-running-without-rc.rst:14]: > I don't see a section marked "Configuring for Emergency RC Receiver Use" below

Yeah, this is a WIP... patches against it welcome.  It requires a chunk of research and *must* be right....

**Comment by amilcarlucas** [comment]
This seams to duplicate this existing page: https://ardupilot.org/copter/docs/common-gcs-only-operation.html

**Comment by Hwurzburg** [comment]
@peterbarker close this?

**Comment by Hwurzburg** [comment]
@peterbarker since there is no response above, I am closing this based on the wiki already addresses this...feel free to reopen if you disagree...

**Pull request closed without merging**